source/azure_iot_private.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License. */
+
+#ifndef AZURE_IOT_PRIVATE_H
+#define AZURE_IOT_PRIVATE_H
+
+#include "azure_iot_result.h"
+
+#include "azure/az_core.h"
+#include "azure/az_iot.h"
```

Comment:
is this header required?

Reply:
not in the header. I put it in the `.c` now